api/v1alpha1/ndb_api_helpers.go

```diff
@@ -269,7 +265,12 @@ func GetActionArgumentsByDatabaseType(databaseType string) (DatabaseActionArgs,
 }
 
 func (m *MysqlActionArgs) GetActionArguments(dbSpec DatabaseSpec) []ActionArgument {
-	return []ActionArgument{}
+	return []ActionArgument{
+		{
+			Name:  "listener_port",
```

Comment:
Is the set of mandatory ActionArgs documented anywhere (for all the engines )?

Reply:
Not sure about that, I generally use the API Equivalent from the UI to determine the args.